src/components/v2/VaultItemUi/index.tsx

```diff
@@ -0,0 +1,157 @@
+// VaultItemUi
+/** @jsxImportSource @emotion/react */
+import React, { useMemo } from 'react';
+import BigNumber from 'bignumber.js';
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+
+import { useTranslation } from 'translation';
+import {
+  convertWeiToCoins,
+  formatCommaThousandsPeriodDecimal,
+  formatToReadablePercentage,
+} from 'utilities/common';
+import { TokenId } from 'types';
+import { getToken } from 'utilities';
+import { Icon } from '../Icon';
+import { Button } from '../Button';
+import { useStyles } from './styles';
+
+export interface IVaultItemUiProps {
+  tokenId: TokenId;
+  rewardTokenId: TokenId;
+  rewardWei?: BigNumber;
+  userStakedWei: BigNumber;
+  stakingAprPercentage: number;
+  dailyEmissionWei: BigNumber;
+  totalStakedWei: BigNumber;
+  onClaim: () => void;
+  onStake: () => void;
+  onReward: () => void;
+  className?: string;
+}
+
+export const VaultItemUi = ({
+  tokenId,
+  rewardTokenId,
+  rewardWei,
+  userStakedWei,
+  stakingAprPercentage,
+  dailyEmissionWei,
+  totalStakedWei,
+  onClaim,
+  onStake,
+  onReward,
+  className,
+}: IVaultItemUiProps) => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+
+  const dataListItems = useMemo(
+    () => [
+      {
+        title: t('vaultItemUi.stakingApr', { stakeTokenName: getToken(tokenId).symbol }),
+        value: formatToReadablePercentage(stakingAprPercentage),
+      },
+      {
+        title: t('vaultItemUi.dailyEmission'),
+        value: (
+          <>
+            <Icon css={classes.tokenIcon} name={rewardTokenId} />
+            {formatCommaThousandsPeriodDecimal(
```

Comment:
You don't need to use `formatCommaThousandsPeriodDecimal` here, `convertWeiToCoins` already uses it internally.